src/www/ui_components/server_list.ts

```diff
@@ -0,0 +1,98 @@
+/*
+  Copyright 2021 The Outline Authors
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+       http://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import {computed, customElement, property} from '@polymer/decorators';
+import {html, PolymerElement} from '@polymer/polymer';
+
+import {Server} from '../model/server';
+
+// TODO (daniellacosse): Migrate server-card.js to TS and export the type from there
+interface ServerCard extends Element {
```

Comment:
Alternatively I could export a JS object in `server-card.js` with JSDoc annotations? Would that be preferred?

Reply:
This is also addressed in the blocked PR - https://github.com/Jigsaw-Code/outline-client/pull/1105

Reply:
That's fine.